behave_django/management/commands/behave.py

```diff
@@ -21,45 +21,63 @@ def add_command_arguments(parser):
         help="Don't create a test database. USE AT YOUR OWN RISK!",
     )
     parser.add_argument(
-        '--keepdb',
+        '-k', '--keepdb',
         action='store_true',
         default=False,
         help="Preserves the test DB between runs.",
     )
 
 
-def add_behave_arguments(parser):
+def add_behave_arguments(parser):  # noqa
     """
     Additional command line arguments extracted directly from behave
     """
 
+    # Option strings that conflict with Django
     conflicts = [
         '--no-color',
-        '--version'
+        '--version',
+        '-c',
+        '-k',
+        '-v',
     ]
 
+    parser.add_argument(
+        'paths',
+        action='store',
+        nargs='*',
+        help="Feature directory, file or file location (FILE:LINE)."
+    )
+
     for fixed, keywords in behave_options:
-        # TODO: accept short options too
         keywords = keywords.copy()
-        long_option = None
-        for option in fixed:
-            if option.startswith("--"):
-                long_option = option
-                break
 
-        # Do not add conflicting options
-        if long_option in conflicts:
+        # Configfile only entries are ignored
+        if not fixed:
             continue
 
-        if long_option:
-            # type isn't a valid keyword for make_option
-            if hasattr(keywords.get('type'), '__call__'):
-                del keywords['type']
-            # config_help isn't a valid keyword for make_option
-            if 'config_help' in keywords:
-                del keywords['config_help']
+        # Build option strings
+        # Conflicting option strings get prefixed with `--behave`
```

Comment:
How about moving this specific comment two lines down? -- We've got a superabundance of comments here.